index.html

```diff
@@ -15,6 +15,7 @@
     <script async src="https://www.w3.org/Tools/respec/respec-w3c" class="remove"></script>
     <script class="remove">
       var respecConfig = {
+		darkMode: true,
```

Comment:
Use spaces not tabs (globally, e.g. in local.css below also)

Reply:
Thanks. I've updated `index.html`.

Regarding `local.css`, it seems like most of it uses tabs. I don't have a strong opinion (but tend to use spaces for personal projects), but if we decide to use spaces, maybe we should document it in https://www.w3.org/International/i18n-activity/guidelines/editing .

Reply:
Why convert tabs to spaces?

Reply:
Because the rest of the file is spaces. And because it makes it easier to maintain alignments. Whatever we do, we shouldn't mix the two indentation schemes.

Reply:
Most style guides now recommend using spaces, like:

* [Google](https://google.github.io/styleguide/htmlcssguide.html#Trailing_Whitespace)
* [Mozilla](https://protocol.mozilla.org/docs/contributing/css-guide#format)
* [WebKit](https://webkit.org/code-style-guidelines/#indentation-no-tabs)
* [WPT](https://web-platform-tests.org/writing-tests/general-guidelines.html#style-rules)
* [ReSpec](https://github.com/w3c/respec/blob/5f0ab541921ee908047fddcedaeb02171e3cab79/.editorconfig#L4)

That said, I think large-scale modifications to whitespaces also have costs and consistency within a file is more important.